packages/chat-stateful-client/src/ResourceDownloadQueue.test.ts

```diff
@@ -71,7 +71,7 @@ describe('ResourceDownloadQueue api functions', () => {
     const editedMessage = { ...originalMessage };
     editedMessage.content = { message: 'edited message', attachments: secondAttachments };
 
-    const queue = new ResourceDownloadQueue(context, tokenCredential);
+    const queue = new ResourceDownloadQueue(context, { credential: tokenCredential, endpoint: 'endpoint' });
```

Comment:
Let's not use magic text?

Reply:
This is a test file, so we kind of have magic text everywhere. Should we just refactor the whole test file?

Reply:
Extracting this to a const value doesn't seem like much.